core/src/main/scala/org/http4s/util/ByteVectorChunk.scala

```diff
@@ -1,14 +1,13 @@
 package org.http4s
 package util
 
-import fs2.Chunk
-import scala.reflect.{ClassTag, classTag}
+import fs2.{Chunk, MonomorphicChunk}
 import scodec.bits.ByteVector
 
-// This will exist in fs2-1.0.  That version is more optimal, because
-// it has a smarter concatAll.  This will do for now.
-private[http4s] class ByteVectorChunk private (val toByteVector: ByteVector)
-    extends Chunk[Byte] {
+import scala.reflect.{ClassTag, classTag}
+
+final class ByteVectorChunk private (val toByteVector: ByteVector)
```

Comment:
Because it's not actually in the published jar.

The whole `fs2.interop.scodec` package is not included.